.changeset/tasty-avocados-hope.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog': patch
+---
+
+Migrate catalog api to declarative integration system.
```

Comment:
Probably good to mention that this is a new `/alpha` export as well.

Reply:
Done 🙌🏻 : https://github.com/backstage/backstage/pull/20169/commits/b62aaa3a68abf6ce0810961c3195b3c1e1ad5a42